inspirehep/records/api/base.py

```diff
@@ -299,10 +301,61 @@ def get_linked_pids_from_field(self, path):
         full_path = ".".join([path, "$ref"])
         pids = [
             PidStoreBase.get_pid_from_record_uri(rec)
-            for rec in self.get_value(full_path, [])
+            for rec in get_value(data, full_path, [])
```

Comment:
do we even use ``self.get_value``? can we just remove it?

Reply:
It's used in many places, for now we have 14 places where it's used in the code... Using it this way it's more natural than passing data to some util function...